components/eamxx/docs/developer/code_structure.md

```diff
@@ -0,0 +1,154 @@
+# EAMxx Source Code Structure
+
+!!! Info inline "E3SM/components/eamxx/"
+
+    ```
+    .
+    ├── cime_config/
+    ├── cmake/
+    │   └── machine-files/
+    │── CMakeLists.txt
+    │── CTestConfig.cmake
+    ├── docs/
+    ├── mkdocs.yml
+    ├── README.md
+    ├── scripts/
+    │   ├── atmchange
+    │   ├── atmquery
+    │   └── test-all-eamxx
+    ├── src/
+    │   ├── control/
+    │   │   └── atmosphere_driver.<x>pp
+    │   ├── diagnostics/
+    │   ├── dynamics/
+    │   │   └── homme/
+    │   ├── mct_coupling/
+    │   ├── physics/
+    │   │   ├── cld_fraction/
+    │   │   ├── cosp/
+    │   │   ├── iop_forcing/
+    │   │   ├── mam/
+    │   │   ├── ml_correction/
+    │   │   ├── nudging/
+    │   │   ├── p3/
+    │   │   ├── register_physics.hpp
+    │   │   ├── rrtmgp/
+    │   │   ├── share/
+    │   │   │   ├── physics_constants.hpp
```

Comment:
Let's remove these files (as well as the atm driver one above). It seems confusing to list some files in some folders  but not all.